x-pack/plugins/apm/tsconfig.json

```diff
@@ -96,7 +96,8 @@
     "@kbn/logs-shared-plugin",
     "@kbn/unified-field-list",
     "@kbn/discover-plugin",
-    "@kbn/observability-ai-assistant-plugin"
+    "@kbn/observability-ai-assistant-plugin",
+    "@kbn/apm-plugin"
```

Comment:
Can you remove this. Not sure why the CI added it. This will cause circular dependency error and fail the build

Reply:
CI added it because lint_ts_projects ran before ESLint, hehe. ESLint changed this line:

`import { Transaction } from "@kbn/apm-plugin/typings/es_schemas/ui/transaction";`